test/integration/acceptance/set_account_detail_test.cpp

```diff
@@ -147,3 +167,69 @@ TEST_F(SetAccountDetail, WithGrantablePerm) {
         ASSERT_EQ(block->transactions().size(), 1);
       });
 }
+
+/**
+ * C276
+ * @given a user with required permission
+ * @when execute tx with SetAccountDetail command with max key
+ * @then there is no tx in proposal
+ */
+TEST_F(SetAccountDetail, HugeKey) {
```

Comment:
You have two test cases: `HugeKey` and `HugeKeyValue`.
The meaning of "huge" is never defined. Does it mean huge but still valid or not?

Surprisingly, here "huge" is a big valid value, but in `HugeKeyValue` test case it is not.

Please rename both test cases with more clear names.